### DISCUSSION TITLE
Added: LDM notes for June 15, 2020

### DISCUSSION THREAD

**Discussion by SingleAccretion** [dotnet/csharplang#3588] [General]
[C# Language Design Notes for June 15, 2020](https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-06-15.md)

Records:
1\. `modreq` for init accessors
2\. Initializing `readonly` fields in same type
3\. `init` methods
4\. Equality dispatch
5\. Confirming some previous design decisions
6\. `IEnumerable.Current`

**Comment by HaloFour** [comment]
I'm curious if there will be an effort to also standardize how the compilers handle a property accessor with a `modreq` modifier?  It's quite concerning that C#, VB.NET and F# all handle the `modreq(IsExternalInit)` differently.  Is the behavior here not a part of the CLR spec?

I'll make the wild assumption that F# will be updated in time to properly respect the `modreq`, but will anything prevent an older version of F# from targeting the newer runtime?
